app/models/manager_refresh/inventory_object.rb

```diff
@@ -93,6 +97,18 @@ def self.add_attributes(inventory_object_attributes)
       end
     end
 
+    def ==(other)
```

Comment:
This seems like an ineffective check, also I don't see it used anywhere in the parser? So maybe we don't need this one?

If we would need it, it should be enough to compare manager_uuid, which is used for indexing @data_index. That one needs to be unique, so we can use it for quick comparing.